cmd/memcached_exporter/main.go

```diff
@@ -27,15 +27,19 @@ import (
 	"github.com/prometheus/common/promlog"
 	"github.com/prometheus/common/promlog/flag"
 	"github.com/prometheus/common/version"
-	"github.com/prometheus/memcached_exporter/pkg/exporter"
+	"github.com/prometheus/exporter-toolkit/web"
+	webflag "github.com/prometheus/exporter-toolkit/web/kingpinflag"
```

Comment:
nit: why not just importing the package as `kingpinflag` or even renaming it to `kingpinwebflag` / `webflagkingpin` instead of asking each package user to provide the most descriptive name as alias?

Reply:
This is coming from prometheus where we also have promlog flag.

Reply:
Thanks for the info.